app/views/partials/attachments_list_atar.scala.html

```diff
@@ -0,0 +1,93 @@
+                    <th id="attachment-details" class="govuk-table__header" scope="col">Details</th>
+                    <th id="attachment-status" class="govuk-table__header" scope="col">Status</th>
+                    <th><span class="visually-hidden">attachment action</span></th>
+                </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+                    @for((file: StoredAttachment, index: Int) <- attachments.zipWithIndex) {
+                        <tr id="@{id}-row-@{index}" class="govuk-table__row">
+                            <span class="pre-wrap"></span>
+                            <td id="@{id}-row-@{index}-title" class="govuk-table__cell">
+                                @attachment_atar(s"$id-row-$index", file, useTabIndex = startAtTabIndex.map(x => x+1+(0*index)))
+                            </td>
+                        </span>
+                                <td id="@{id}-row-@{index}-uploaded_by" class="govuk-table__cell w-15">
+                                    @if(file.operator.isDefined) {
+                                        @file.operator.flatMap(_.name).getOrElse("Unknown")
+                                    } else {
+                                    @c.application.contact.name
+                                    }
+                                </td>
+
+                            <td id="@{id}-row-@{index}-attachment-details" class="govuk-table__cell w-15">
+                                @if(file.description.isDefined) {
+                                     @file.description
+                                }
+                            </td>
+
+                            <td id="@{id}-row-@{index}-attachment-status" class="govuk-table__cell w-15">
+                                <span class="visually-hidden">attachment-status</span>
+                                @if(file.scanStatus.contains(ScanStatus.READY)) {
+                                  @{(file.public, file.shouldPublishToRulings) match {
+                                          case (false, _) =>
+                                              <span id="@{id}" class="govuk-tag govuk-tag--yellow"> {messages("case.attachment.upload.status-confidential")}</span>
+                                          case (true, true) =>
+                                              <span id="@{id}" class="govuk-tag govuk-tag--green"> {messages("case.attachment.upload.status-published")}</span>
+                                          case (true, false) =>
+                                              <span id="@{id}" class="visually-hidden"> </span>
+                                      }
+                                  }
+                                } else {
+                                    <span id="@id" class="govuk-tag govuk-tag--red">@messages("case.attachment.upload.status-failed")</span>
```

Comment:
Does this mean it will say `upload failed` if the scan has not finished yet?

Reply:
Good spot! We have refactored the logic.